.github/workflows/ci.yml

```diff
@@ -41,69 +42,44 @@ jobs:
     - name: Install python dependencies
       run: |
         pip install --upgrade pip
-        pip install -r requirements.txt
-        # Change plugin_name to test we're not specific to "aiida-diff"
-        #cookiecutter --no-input . plugin_name=${PLUGIN_NAME}
-        #pip install -e ${PLUGIN_NAME}[testing]
-      env:
-        PLUGIN_NAME: aiida-mlip
+        pip install -e .[testing]
 
     - name: Run test suite
       env:
         # show timings of tests
         PYTEST_ADDOPTS: "--durations=0"
-        PLUGIN_NAME: aiida-mlip
-      run: |
-        cd ${PLUGIN_NAME}
-        pytest -v
+      run: pytest --cov aiida_mlip --cov-append .
 
   docs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.10
+    - name: Set up Python 3.11
       uses: actions/setup-python@v2
       with:
-        python-version: "3.10"
+        python-version: 3.11
     - name: Install python dependencies
-      env:
-        PLUGIN_NAME: aiida-mlip
       run: |
         pip install --upgrade pip
-        pip install -r requirements.txt
-        # Change plugin_name to test we're not specific to "aiida-diff"
-        #cookiecutter --no-input . plugin_name=${PLUGIN_NAME}
-        #pip install -e ${PLUGIN_NAME}[docs]
+        pip install -e .[docs]
     - name: Build docs
-      env:
-        PLUGIN_NAME: aiida-mlip
-      run: cd ${PLUGIN_NAME}/docs && make
+      run: cd docs && make
 
   pre-commit:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.10
+    - name: Set up Python 3.11
       uses: actions/setup-python@v2
       with:
-        python-version: "3.10"
+        python-version: "3.11"
     - name: Install python dependencies
-      env:
-        PLUGIN_NAME: aiida-mlip
       run: |
         pip install --upgrade pip
-        pip install -r requirements.txt
-        # Change plugin_name to test we're not specific to "aiida-diff"
-        #cookiecutter --no-input . plugin_name=${PLUGIN_NAME}
-        #pip install -e ${PLUGIN_NAME}[pre-commit,docs,testing]
+        pip install -e .[pre-commit,docs,testing]
```

Comment:
here I will you 
python -m pip install

Reply:
> here I will you python -m pip install

This will likely change assuming we're happy with the poetry changes anyway, but to check, are you just suggesting `pip install` -> `python -m pip install`, rather than getting rid of the square brackets? (I think we need `[pre-commit]` to make sure `pre-commit install` etc. runs, I'm not actually sure we need the other two here)

Reply:
yes I have still traumas after fixing things because pip was the wrong one... so always python -m pip instead of pip alone